1saNe/flasktodoapp
# add: Readme and requirements.txt

# Pull Request Description

This pull request adds the missing information to the project, including:
- A brief explanation of the features in the application
- A list of prerequisites required to run the application
- Detailed steps on how to run the application
- A `requirements.txt` file to specify the required packages and their versions

All of these additions will help make it easier for other developers to run and understand the project.

# How to test

1. Clone the repository
2. Navigate to the repository
3. Set up a virtual environment and activate it
4. Install the required packages from the `requirements.txt` file
5. Run the application
6. Verify that all of the features work as expected and that the information in the README file is accurate and helpful.
